src/components/header/dropdownMenu.js

```diff
@@ -51,24 +50,23 @@ const DropdownMenu = ({ teams, selectedTeam, selectedTeamColor, getSelectedTeam
     const isSelected = selectedTeamId === teamId;
 
     return (
-      !isSelected && (
-        <div
-          style={{ backgroundColor: primaryColor }}
-          className={classnames(styles.optionContainer, isSelected && styles.displayNone)}
-          ref={innerRef}
-          {...innerProps}
-        >
-          <div className={styles.optionImgContainer}>
-            <img
-              src={`https://cdn.nba.net/assets/logos/teams/secondary/web/${tricode}.svg`}
-              title="team logo"
-              alt="team logo"
-              className={styles.optionImage}
-            />
-          </div>
-          <div className={styles.optionTitle}>{ttsName}</div>
-        </div>
-      )
+      !isSelected &&
+      teamsArray.map((team) => {
+        return (
+          team.TRI_CODE === tricode && (
+            <div
```

Comment:
Something odd is happening in the dropdown. There is a space after last team item:

<img width="329" alt="Screen Shot 2020-06-07 at 12 40 46 PM" src="https://user-images.githubusercontent.com/623755/83976357-726e2200-a8c7-11ea-8d12-236485ab9aa5.png">